packages/rancher-gatekeeper/generated-changes/patch/templates/gatekeeper-admin-serviceaccount.yaml.patch

```diff
@@ -1,8 +0,0 @@
---- charts-original/templates/gatekeeper-admin-serviceaccount.yaml
-+++ charts/templates/gatekeeper-admin-serviceaccount.yaml
-@@ -8,4 +8,4 @@
-     heritage: '{{ .Release.Service }}'
-     release: '{{ .Release.Name }}'
-   name: gatekeeper-admin
--  namespace: gatekeeper-system
-+  namespace: '{{ .Release.Namespace }}'
```

Comment:
The configurable namespace was added upstream https://github.com/open-policy-agent/gatekeeper/pull/1233/files#diff-3e3fa1de58e947aa0449a35c79a41661bc3bf27e4c7186e9d8df42fd8dc13031